docs/setup/install-existing.md

```diff
@@ -107,22 +149,161 @@ The [aad-pod-identity](https://github.com/Azure/aad-pod-identity) gives a clean
+    # This blacklist all hostnames and paths, prohibiting AGIC from applying config for any of them.
+    # Use "kubectl get AzureIngressProhibitedTargets" to view and change prohibited targets
+    brownfield:
+        enabled: false
     ```
 
-    **NOTE:** The `<identity-resource-id>` and `<identity-client-id>` are the properties of the Azure AD Identity you setup in the previous section. You can retrieve this information by running the following command:
+    **NOTE:** The `<identity-resource-id>` and `<identity-client-id>` are the properties of the Azure AD Identity you
+    setup in the previous section. Run the following command to obtain these values:
 
     ```bash
     az identity show -g <resourcegroup> -n <identity-name>
     ```
 
     Where `<resourcegroup>` is the resource group in which AKS cluster is running (this would have the prefix `MC_`).
 
-1. Install the helm chart `application-gateway-kubernetes-ingress` with the `helm-config.yaml` configuration from the previous step
+1. Install Helm chart `application-gateway-kubernetes-ingress` with the `helm-config.yaml` configuration from the previous step
 
     ```bash
     helm install -f <helm-config.yaml> application-gateway-kubernetes-ingress/ingress-azure
     ```
 
+    Alternatively you can combine the `helm-config.yaml` and the Helm command in one step:
+    ```bash
+    helm install ./helm/ingress-azure \
+         --name ingress-azure \
+         --version 0.7.1 \
+         --namespace default \
+         --debug \
+         --set appgw.name=applicationgatewayABCD \
+         --set appgw.resourceGroup=your-resource-group \
+         --set appgw.subscriptionId=subscription-uuid \
+         --set armAuth.type=servicePrincipal \
+         --set armAuth.secretJSON=$(az ad sp create-for-rbac --subscription <subscription-uuid> --sdk-auth | base64 -w0) \
+         --set rbac.enabled=true \
+         --set verbosityLevel=3 \
+         --set kubernetes.watchNamespace=default \
+         --set aksClusterConfiguration.apiServerAddress=aks-abcdefg.hcp.westus2.azmk8s.io \
+         --set brownfield.enabled=false
+    ```
+
 1. Check the log of the newly created pod to verify if it started properly
 
-Refer to the [tutorials](../tutorial.md) to understand how you can expose an AKS service over HTTP or HTTPS, to the internet, using an Azure Application Gateway.
+Refer to the [tutorials](../tutorial.md) to understand how you can expose an AKS service over HTTP or HTTPS, to the internet, using an Azure App Gateway.
+
+## Hybrid Environment
+AGIC will assume full ownership of the AGIC configuration. What if we'd like to manually add a listener and a backend?
+We have the option to constrain AGIC, so it controls only a subset of the App Gateway's properties.
+
+### Scenario
+Let's take an imaginary App Gatway, which manages traffic for 2 web sites:
+  - prod.contoso.com
+  - dev.contoso.com
+
+We need to:
```

Comment:
should we mention what version support sthis feature ?

Reply:
Not a bad idea - adding...